examples/SignMultiple.purs

```diff
@@ -21,6 +21,7 @@ import Contract.Test.E2E (publishTestFeedback)
 import Contract.Transaction
   ( BalancedSignedTransaction
   , TransactionHash
+  , TxOutRefCache
```

Comment:
We definitely should not expose `TxOutRefCache`. Let's remove logging of locked outputs (it's not good to expose the internals to the user anyway, and we don't have any assertions here).